streams-tests/shared/src/test/scala/zio/stream/ZPipelineSpec.scala

```diff
@@ -7,9 +7,23 @@ import zio.test._
 import scala.io.Source
 
 object ZPipelineSpec extends ZIOBaseSpec {
-
+  def utf8EncodePipeline = {
```

Comment:
Could you just embed this in the `test` instead of having it extracted here?

Reply:
```scala
 suite("utf8Encode")(
        test("test1") {
          val expected: List[Byte] = List(49, 50)
          for {
            actual <- ZStream.fromChunks(Chunk.single("1"), Chunk.single("2"))
              .via(ZPipeline.utf8Encode)
              .run(ZSink.collectAll)
              .map(_.toList)
          } yield assert(actual)(equalTo(expected))
        }
```
I changed it